index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="sv">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <link rel="stylesheet" href="./lucky.style.css">
+ <link rel="preconnect" href="https://fonts.googleapis.com">
+ <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+ <link href="https://fonts.googleapis.com/css2?family=Abyssinica+SIL&family=Roboto+Condensed:ital@1&display=swap" rel="stylesheet">
+ <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Sofia">
+ <title>Lucky Duck</title>
+</head>
+<body>
+ <div class="Navbar">
+  <div class="logo"></div>
+  <div class="name">
```

Comment:
I think having the text takes up too much space. Just having their logo in the top right would be better, since it needs to fit the other stuff :)

Reply:
But it gives a beautiful shape next to the logo in general, think about it tomorrow, and if you don't want it, I will remove it.

Reply:
What if we make it so that if the other items grow larger, it will hide the text logo and only show the square image logo. We are required by the spec to use the square logo somewhere on the page.